gpuarray/include/gpuarray/buffer_blas.h

```diff
@@ -1,12 +1,12 @@
-#ifndef GPUARRAY_BUFFER_BLAS_H
-#define GPUARRAY_BUFFER_BLAS_H
+#ifndef LIBGPUARRAY_GPUARRAY_BUFFER_BLAS_H
+#define LIBGPUARRAY_GPUARRAY_BUFFER_BLAS_H
```

Comment:
I deleted some things that I wrote by mistake. I meant to write the following, sorry:  
"Google style advices for a header guard like this:
`#ifndef <project name>_<directories>_<file name>`. [...]"